measure-builds/src/test/java/com/automattic/android/measure/BuildTimePluginTest.kt

```diff
@@ -138,6 +138,7 @@ class BuildTimePluginTest {
             assertThat(executionData.executedTasks).hasSize(1).first().satisfies({ task ->
                 assertThat(task.name).isEqualTo(":help")
                 assertThat(task.state).isEqualTo(MeasuredTask.State.EXECUTED)
+                assertThat(task.endTimestamp - task.startTimestamp).isEqualTo(task.duration.inWholeMilliseconds)
```

Comment:
Arguably, this assertion shouldn't be part of this test in particular, but it was quite convenient. But I can write a new dedicated test if you insist.